lib/sitemap/sitemap_presenter.rb

```diff
@@ -17,7 +19,11 @@ def last_updated
     return nil unless document.public_timestamp
 
     # Attempt to parse timestamp to validate it
-    DateTime.iso8601(document.public_timestamp)
+    parsed_date = DateTime.iso8601(document.public_timestamp)
+
+    # Ignore timestamps for documents backdated to before GOV.UK was launched
+    return nil if parsed_date < GOVUK_LAUNCH_DATE
```

Comment:
Yeah, Mat suggested this as well because it would stop search engines from making assumptions about the recency of the pages.

I've made the change. 👍